File: JavaScript_Courses/JavaScript_Basics/Preparation/02_Conditional_Statements_Exercise/01_Sum_Seconds.js
function sumSeconds(input) {
    let secondsFirst = Number(input[0]);
    let secondsSecond = Number(input[1]);
    let secondsThird = Number(input[2]);

    let sumSeconds = secondsFirst + secondsSecond + secondsThird;
    let minutes = Math.floor(sumSeconds / 60);
    let seconds = sumSeconds % 60;

    if (seconds < 10) {
        console.log(`${minutes}:0${seconds}`);
    } else {
        console.log(`${minutes}:${seconds}`);
    }
}

// sumSeconds(["35", "45", "44"]);

File: Python_Courses/Python_Basics/Python_PB_2024_10/02_Conditional_Statements_Exercise/01_Sum_Seconds.py
seconds_first = int(input())
seconds_second = int(input())
seconds_third = int(input())

total_seconds = seconds_first + seconds_second + seconds_third  # 121
minutes = total_seconds // 60  # 121 // 60 == 2
seconds = total_seconds % 60  # 121 % 60 == 1

print(f'{minutes}:{seconds :02d}')
